client/src/components/MuiFooter.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import {
+    // Container,
```

Comment:
are you going to be using this in the future `Container`? If not, let's remove it.

Reply:
Removed old footer component.